index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <meta name="description" content="An aspiring software developer with experience in desktop, web & mobile applications.">
+        <title>Portfolio</title>
+        <link rel="stylesheet" href="styles.css">
+    </head>
+    <body>
+        <header>
+            <div id="time-utility">
+                <div>
+                    <img src="./icons/time-slot.png">
+                </div>
+                <div id="utilities">
+                    <div>
+                        <img src="./icons/cellular.png">
+                    </div>
+                    <div>
+                        <img src="./icons/Wifi.png">
+                    </div>
+                    <div>
+                        <img src="./icons/Battery.png">
+                    </div>
+                </div>
```

Comment:
- [ ] From the project design, the `time`, `battery`, and `wifi` were not meant to be added to the project but to show you what the mobile version looks like. So please kindly remove them from your project. :pray:
![Screenshot from 2022-10-27 16-32-29](https://user-images.githubusercontent.com/92303891/198349532-7f05ea67-6368-4f60-bb9a-99e3850f9eec.png)